docs/human-interface-guidelines.md

```diff
@@ -332,27 +332,27 @@ The indicator area is notorious for becoming cluttered and having inconsistent b
 * **The application is for IM, IRC, e-mail, news-reading, or music playback.** Instead, integrate the application with the existing messaging or sound menus.
 
 ---
-See also: [Farewell To The Notification Area](http://design.canonical.com/2010/04/notification-area/) by Matthew Paul Thomas
+See also: [Farewell to the Notification Area](http://design.canonical.com/2010/04/notification-area/) by Matthew Paul Thomas
 
 # Container Widgets {#container-widgets}
 
 ## Windows {#windows}
 
-Windows are the foundation that your app is built on. They provide a sort of canvas with basic actions built in such as closing an app, resizing it, etc. Although users may see all windows as being the same, there are several distinct types of windows in elementary OS. It's important to understand the types of windows available to you, window behavior in general, and behavior that is specific to a certain window type. This section will cover the different types of windows available in elementary OS. Although each type of window is a bit different, think of them all as a subclass of a window. Unless otherwise stated, they all behave as an average window.
+Windows form the foundation of your app. They provide a canvas with basic, built-in actions such as "closing" and "resizing". Although users may see windows as being all the same, elementary OS has several distinct window types. It's important to understand the types of windows available to you, window behavior in general, and behavior that is specific to a certain window type. This section will cover the different types of windows available in elementary OS. Although each type of window is a bit different, think of them all as a subclass of a window. Unless otherwise stated, they all behave as an average window.
 
-* **View Windows** are the most literal kind of "Window" available. They show content such as presentations, web pages, documents, etc. These Windows act as a sort of frame around the specific content they display. They typically allow multiple instances of an application to be shown and may utilize tabs to help keep content organized without displaying too many windows.
-* **App Windows** are the main windows of apps that are not document-based. These windows typically only allow one instance of an application to be shown.
+* **View Windows** are the most literal kind of "Window" available. They show content such as presentations, web pages, documents, etc. These Windows act as a sort of frame around their specific, displayed content. They typically afford the option of opening multiple instances of an application, and may use tabs to structure an overabundance of windows.
```

Comment:
More looking to avoid passive voice than trying to force a change for "allow" ("let" is always an easy substitution and windows technically don't have agency to allow but they can afford, especially since this is a design doc). I think part of the issue is the original copy—multiple instances feels like it's implied but I tried to work it in anyway. Rephrased